src/main/java/training/day5/School.java

```diff
@@ -0,0 +1,152 @@
+package training.day5;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+class School {
+    ArrayList<Student> students;
+    ArrayList<Teacher> teachers;
+
+    public School() {
+        students = new ArrayList<>();
+        teachers = new ArrayList<>();
+    }
+
+    public void addStudent(Student student) {
+        students.add(student);
+    }
+
+    public void addTeacher(Teacher teacher) {
+        teachers.add(teacher);
+    }
+
+
+    public void printInformation() {
```

Comment:
That is good. You can read about "toString()" method and try to refactor this method with it.